apps/fulfillment/.env.netlify-test

```diff
@@ -0,0 +1,2 @@
+# ORYX_FULFILLMENT_MOCK_PROXY_URL=https://backend-api.de.demo-picking-app.cloud.spryker.toys/
+# ORYX_FULFILLMENT_MOCK_BASE_PATH=/.netlify/functions/server
```

Comment:
Do we need this variable? It seems like we are not using it

Reply:
For now we are not proxying to real bapi because login is not implemented yet but we will enable it once that is done.

Reply:
Actually one is required now so I've uncommented it but the other for now is not needed.